### ISSUE TITLE
False statement in warning

### ISSUE THREAD

**Issue by EmmanuelBeziat** [shinnn/vscode-stylelint#58]
Hi,

I just saw this warning:

```
[css] Property is ignored due to the display. With 'display: inline-block', float properties have no effect.
```

This is untrue: the only "display" value that would override a float statement is "none". Any other case is the opposite: the display property has no effect on float elements.

Here’s a test: https://jsfiddle.net/wr5pgn1c/1/ 

Thanks!

**Comment by shinnn** [comment]
The warning you pasted is not from this extension, but from the VSCode built-in CSS linter. Open this issue on https://github.com/Microsoft/vscode/issues instead.

**Comment by EmmanuelBeziat** [comment]
Thanks.
